src/sqlancer/MainOptions.java

```diff
@@ -50,6 +50,21 @@ public class MainOptions {
     @Parameter(names = "--print-failed", description = "Logs failed insert, create and other statements without results", arity = 1)
     private boolean loggerPrintFailed = true; // NOPMD
 
+    @Parameter(names = "--qpg-enable", description = "Enable the experimental feature Query Plan Guidance (QPG)", arity = 1)
+    private boolean enableQPG;
+
+    @Parameter(names = "--qpg-log-query-plan", description = "Logs the query plans of each query (requires --qpg-enable)", arity = 1)
+    private boolean logQueryPlan;
+
+    @Parameter(names = "--qpg-max-interval", description = "The maximum number of iterations to mutate tables if no new query plans (requires --qpg-enable)")
+    private static int qpgMaxInterval = 1000;
+
+    @Parameter(names = "--qpg-reward-weight", description = "The weight (0-1) of last reward when updating weighted average reward (requires --qpg-enable)")
+    private static double qpgk = 0.25;
+
+    @Parameter(names = "--qpg-selection-probability", description = "The probability (0-1) of the random selection of mutators (requires --qpg-enable)")
```

Comment:
For these parameters, can we add one sentence describing the implications of setting a lower/higher value? Here, we could mention something like the following: `A higher value favors exploration over exploitation.`